web/pgadmin/__init__.py

```diff
@@ -335,7 +335,11 @@ def get_locale():
     ##########################################################################
     if config.CONFIG_DATABASE_URI is not None and \
             len(config.CONFIG_DATABASE_URI) > 0:
-        app.config['SQLALCHEMY_DATABASE_URI'] = config.CONFIG_DATABASE_URI
+        uri = config.CONFIG_DATABASE_URI
+        pg_pattern = r'^postgresql(\+psycopg3)?:'
```

Comment:
Can you move this logic to a common function probably in the file `web/pgadmin/utils/__init__.py` and then use it at both places?

Reply:
@adityatoshniwal done and added doc changes. ok?